database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
Not knowing your thought process behind the schema design, I'd flip the question back: why would you _not_ want a foreign key?

IMHO, if you expect to be writing SQL like (this is an approximation, not actual code):

```
SELECT * FROM organizations
JOIN users
ON organizations.user_id = users.id
WHERE user.id = <LOGGED_IN_USER_ID>
```

Then it makes sense to be explicit about the foreign-key-ness of the column.